processing-status-api-function-app/src/main/kotlin/gov/cdc/ocio/processingstatusapi/functions/status/GetUploadStatusFunction.kt

```diff
@@ -41,21 +41,20 @@ class GetUploadStatusFunction(
     /**
      * Advanced query for dex uploads, including sorting, filtering and pagination.
      *
-     * @param destinationId String
+     * @param dataStreamId String
      * @param stageName String
      * @return HttpResponseMessage
      */
-    fun uploadStatus(destinationId: String, stageName: String): HttpResponseMessage {
+    fun uploadStatus(dataStreamId: String, stageName: String): HttpResponseMessage {
 
-        logger.info("destinationId = $destinationId")
+        logger.info("dataStreamId = $dataStreamId")
 
         val dateStart = request.queryParameters["date_start"]
         val dateEnd = request.queryParameters["date_end"]
         val pageSize = request.queryParameters["page_size"]
         val pageNumber = request.queryParameters["page_number"]
-        val extEvent = request.queryParameters["ext_event"]
-        val sortBy = request.queryParameters["sort_by"]
```

Comment:
Would you please put the sort_by and sort_order back in?   We'll need that when we fix sorting.  Might as well leave that code here.